tests/subsys/emds/emds_api/src/main.c

```diff
@@ -199,13 +226,20 @@ static bool pragma_clear_flash(const void *s)
 	return (*state == EMDS_TS_CLEAR_FLASH);
 }
 
-static bool pragma_flash_no_store(const void *s)
+static bool pragma_no_store(const void *s)
 {
 	const enum test_states *state = s;
 
 	return (*state == EMDS_TS_NO_STORE);
 }
 
+static bool pragma_several_store(const void *s)
+{
+	const enum test_states *state = s;
+
+	return (*state == EMDS_TS_SEVERAL_STORE);
```

Comment:
```suggestion
	return *state == EMDS_TS_SEVERAL_STORE;
```
Is the brackets neccesary here?